### ISSUE TITLE
Auto-peering

### ISSUE THREAD

**Issue by deavmi** [deavmi/figNET#6]
Once one node has at least one peer then let him get a list of peers that the remote peer (which he has already peered with) has and then add those peers to his own list.

**Comment by deavmi** [comment]
Each node has their own `nodes.list` file but there should also be a public list of nodes stored on the network stored in a folder that should be called `netInfo`.

Hence we must re-write some code and make the webserver serve a folder called `network/` and within there have the folders: `news/` and `netInfo/`,

**Comment by Walkman100** [comment]
Can't you just make the node's `nodes.list` file public?

**Comment by deavmi** [comment]
No. Then people could change the contents of that file and you could loose your nodes.

**Comment by Walkman100** [comment]
I mean publicly readable

**Comment by Walkman100** [comment]
via http

**Comment by deavmi** [comment]
Well. We could show on the satidctics page the nkdes that the node is using.

**Comment by Walkman100** [comment]
also then the command line program can read it via http instead of having to make a new folder to sync
